backend/app/views/spree/admin/return_authorizations/_form.html.erb

```diff
@@ -1,4 +1,5 @@
 <% allow_return_item_changes = !@return_authorization.customer_returned_items? %>
+<% stock_locations = Spree::StockLocation.order_default.active %>
```

Comment:
would it make sense to pull this into the controller? same for the reimbursements/edit form